internal/oc/release.go

```diff
@@ -148,17 +151,33 @@ func (r *release) GetMajorMinorVersion(log logrus.FieldLogger, releaseImage stri
 
 func (r *release) GetReleaseArchitecture(log logrus.FieldLogger, releaseImage string, releaseImageMirror string, pullSecret string) (string, error) {
 	var cmd string
+	var cmdMultiarch string
 	if releaseImage == "" && releaseImageMirror == "" {
 		return "", errors.New("no releaseImage nor releaseImageMirror provided")
 	}
 	if releaseImageMirror != "" {
 		cmd = fmt.Sprintf(templateImageInfo, releaseImageMirror)
+		cmdMultiarch = fmt.Sprintf(templateSkopeoDetectMultiarch, releaseImageMirror)
 	} else {
 		cmd = fmt.Sprintf(templateImageInfo, releaseImage)
+		cmdMultiarch = fmt.Sprintf(templateSkopeoDetectMultiarch, releaseImage)
 	}
```

Comment:
```suggestion
        image := releaseImageMirror
        if image == "" {
                image = releaseImage
        }
	if image == "" {
		return "", errors.New("no releaseImage nor releaseImageMirror provided")
	}
	cmd := fmt.Sprintf(templateImageInfo, image)
	cmdMultiarch := fmt.Sprintf(templateSkopeoDetectMultiarch, image)
```

Feels simpler to me even if it isn't a whole lot shorter :man_shrugging: You can take it or leave it how it is, I don't feel all that strongly about it.

Reply:
I will handle it as part of https://issues.redhat.com/browse/MGMT-11494 because we anyway want to extend a logic of this function